tasks/farm5/ImportTasks.wdl

```diff
@@ -96,3 +96,43 @@ task BatchSplitUpInputFile {
   }
 
 }
+
+
+# Check that there are mandatory columns: sample_id, irods_path
+# Check that each lanelet row has the same sample ID
+task ValidatePerSampleManifestFile {
+  input {
+    File per_sample_manifest_file
+
+    String docker_tag = "sangerpathogens/malaria-irods@sha256:adadaf506ac1d99dfc9a0eb2d8f4cad4527e1a0d00fbf18d8864155ce16038d8"
+    Int num_cpu = 1
+    Int memory = 3000
+    String? lsf_group
+    String? lsf_queue
+    RunTimeSettings runTimeSettings
+  }
+
+  command <<<
+
+  python <<CODE
```

Comment:
I think this should be inset (both for the the CODE block, and then for the embedded python)